bin/cmds/get-config.js

```diff
@@ -0,0 +1,19 @@
+const { echo } = require('shelljs');
+const { get } = require('lodash');
+
+exports.command = 'get-config';
+exports.desc = 'return mdeprc properties';
+exports.builder = async (yargs) => {
+  yargs
+    .option('paths', {
+      describe: 'path in config',
+      type: 'string',
+      array: true,
+      default: [],
+    })
+    .strict(false)
+    .help();
+};
+exports.handler = (argv) => {
+  echo(get(argv, argv.paths));
```

Comment:
if it returns an object - it wont print properly
why not do standard `console.info('%smth', get(....))` ?

Reply:
<img width="511" alt="image" src="https://user-images.githubusercontent.com/54023298/69131783-dc59e200-0ac3-11ea-97be-804c9a5f09c0.png">